1001-2000/LIT1896Martyr.xml

```diff
@@ -1,10 +1,12 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
-schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT1896Martyr">
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">Martyrium of Irenaeus, Bishop of Sirmium</title>
+            <title xml:lang="en" xml:id="t1">Martyrium of Irenaeus, Bishop of Sirmium</title>
+            <title xml:lang="gez" xml:id="t2">ስምዕ፡ ዘሄሬኔዎስ፡ ጳጳስ፡ ዘኤርሶም፡</title>
+            <title xml:lang="gez" type="normalized" corresp="#t2">Sǝmʿ za-Herenewos ṗāṗṗās za-ʾErsom</title>
```

Comment:
ṗāṗās according to BM guidelines (https://betamasaheft.eu/Guidelines/?id=dubious-spelling), except if you object to it?